libobs/obs.h

```diff
@@ -1575,8 +1581,29 @@ EXPORT void obs_scene_atomic_update(obs_scene_t *scene,
 EXPORT void obs_sceneitem_addref(obs_sceneitem_t *item);
 EXPORT void obs_sceneitem_release(obs_sceneitem_t *item);
 
-/** Removes a scene item. */
+/** Removes and adds a scene item. */
```

Comment:
This comment makes it seem like `obs_sceneitem_remove` both removes and adds a scene item.  I'd suggest writing separate comments.